### PR TITLE
update compability chart

### LINKED ISSUE #440: Fix compatibility charts in weave repo as they've fallen a couple versions behind

### PR DESCRIPTION
**Pull request by husio**
resolve #440 

I am not sure about tendermint version so I have just repeated the current one. I do not think we have updated our tendermint requirement.

### CODE REVIEW

**Comment by ruseinov** [comment]
@husio the tendermint version is normally in Gopkg.toml, but we didn't touch it since `0.29.1` - so all good on that front!

**Comment by ruseinov** [comment]
I think we are missing `0.12.1` release for consistency, otherwise looks goo to me!

**Comment by ruseinov** [comment]
Actually, forget it. We can ignore a minor version as it's binary compatible.

File: README.md [hunk 1]
```diff
@@ -86,17 +86,20 @@ Once it compiles, I highly suggest going through the
...
 |v0.10.x | v0.27.4|
 |v0.9.3	| v0.25.0|
 
-| Weave | Protobuf compatible to previous version| Comments | 
+| Weave | Protobuf compatible to previous version| Comments |
 |--------|--------------------|------------------|
-|v0.12.0 | :question:| Currency no longer supports supplying SigFigs nor returns them |
-|v0.11.1 | :heavy_check_mark:||
+|v0.13.0 | :heavy_check_mark:| |
+|v0.12.0 | :x:| Currency no longer supports supplying SigFigs nor returns them. Changes in `x/escrow` |
```
Comment (ethanfrey) [inline README.md]: Changes in x/escrow were in 0.12.1
Change in x/multisig were in 0.13.0

**Pull request merged**